_posts/2021-03-17-cockpit-240.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Cockpit 240
+author: mvo
+date: 2021-03-17
+tags: cockpit linux
+slug: cockpit-240
+category: release
+summary: Cockpit with various improvements
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 240.
+
+## New localization: Norwegian Bokmål
+
+Cockpit is now also available in Norwegian Bokmål localization. Thank
+you Stein-Aksel Basma for this contribution!  Cockpit is being
+translated in [Weblate](https://translate.fedoraproject.org/projects/cockpit/master/)
+where anyone can help to improve Cockpit.
+
+## Performance metrics: Journal integration
+
+For every event on metrics page relevant journal logs are provided to
+help identify why some event has happened.
+
+![Journal entries for a spike](/images/metrics-journal.png)
+
+## Machines: support authentication against cloud images
+
+Distro cloud images have locked login accounts by default. The user
+can now specify the root and user account login credentials when
+creating VMs based on cloud images. The cloud image itself will be
+used as the backing store for the VM's disk.
```

Comment:
The last sentence implies that this feature is much larger than the summary implies.

Which way is it?  We only added authentication for existing cloud-image-as-backing-store VMs, or we added the feature entirely?

Reply:
Why does it imply that the feature is larger :)? It totally is but I just want to see how you figured it out.

Cloud images can be customized through cloud init in much more deep way than just login credentials (ssh keys, network configuration, adding packages is some of them)

We don't plan to add support for none of these, we will just keep it simple and limited to this feature that I describe here.

Reply:
I think we can remove the last sentence. It's not relevant to know what a backing store is to use the new feature.

Reply:
As someone who knows cloud images, I find the description a bit confusing. Maybe s/locked/no/ in the first sentence, dropping the redundant "when creating VMs based on cloud images", and pointing out the vehicle:

| The user can now specify the root and user account login credentials, which will be passed to [cloud-init](https://cloud-init.io/).

This is concise, not too technical, and hints at what's going on for people who actually use cloud images.

Reply:
Done.